src/Microsoft.SqlTools.ServiceLayer/EditData/SmoEditMetadataFactory.cs

```diff
@@ -46,22 +57,23 @@ public EditTableMetadata GetObjectMetadata(DbConnection connection, string objec
 
             // Connect with SMO and get the metadata for the table
             Server server = new Server(new ServerConnection(sqlConn));
+            Database db = new Database(server, sqlConn.Database);
             TableViewTableTypeBase smoResult;
             switch (objectType.ToLowerInvariant())
             {
                 case "table":
-                    smoResult = server.Databases[sqlConn.Database].Tables[objectName];
+                    smoResult = objectNamedParts.Length == 1
+                        ? new Table(db, objectNamedParts[0])                        // No schema provided
+                        : new Table(db, objectNamedParts[1], objectNamedParts[0]);  // Schema provided
```

Comment:
Agreed. This is really just temporary stuff since the code will likely be blown away with the metadata service @kburtram is working on.